packages/react/label/src/Label.tsx

```diff
@@ -78,12 +71,12 @@ const Label = React.forwardRef((props, forwardedRef) => {
 
   return (
     <LabelContext.Provider value={React.useMemo(() => ({ id, ref: labelRef }), [id])}>
-      <Primitive {...labelProps} as={as} selector={selector} ref={ref} id={id} role="label" />
+      <Primitive {...labelProps} as={as} ref={ref} id={id} role="label" />
     </LabelContext.Provider>
   );
 }) as LabelPrimitive;
 
-Label.displayName = 'Label';
+Label.displayName = NAME;
```

Comment:
These changes also nicely highlighted when we were not using the NAME in displayName, so I've updated where necessary.